bin/recog_verify

```diff
@@ -48,15 +55,45 @@ if ARGV.empty?
   exit(1)
 end
 
+def validate(xmlfile_path, xsd_path)
+  schema = Nokogiri::XML::Schema(File.read(xsd_path))
+  doc = Nokogiri::XML(File.read(xmlfile_path))
+  schema.validate(doc)
+end
+
 warnings = 0
 failures = 0
+formatter = Recog::Formatter.new(options, $stdout)
 ARGV.each do |arg|
   Dir.glob(arg).each do |file|
-    ndb = Recog::DB.new(file)
-    verifier = Recog::VerifierFactory.build(options, ndb)
```

Comment:
I don't disagree with the removal of `VerifierFactory`, but would you share your thoughts behind the change?

Reply:
Removes a level of indirection and simplifies the code base. All it does is three things (so it doesn't really do anything special), and it's only used in one place. Additionally, I needed to reuse the formatter to report the XML errors which was previously only created within the factory.